### PR TITLE
feat(migrations): Add migrations for sessions tables

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: snuba/migrations/snuba_migrations/sessions/0001_sessions.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+    def forwards_dist(self) -> Sequence[operations.Operation]:
+        return [
+            operations.CreateTable(
+                storage_set=StorageSetKey.SESSIONS,
+                table_name="sessions_raw_dist",
+                columns=raw_columns,
+                engine=table_engines.Distributed(
+                    local_table_name="sessions_raw_local",
+                    sharding_key=None,  # TODO: what was the sharding key?
```
Comment (fpacifici) [inline snuba/migrations/snuba_migrations/sessions/0001_sessions.py:127]: The sharding key is `org_id`

File: snuba/migrations/snuba_migrations/sessions/0001_sessions.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                ),
+            ),
+            operations.CreateTable(
+                storage_set=StorageSetKey.SESSIONS,
+                table_name="sessions_hourly_dist",
+                columns=aggregate_columns,  # TODO: needs verification
+                engine=table_engines.Distributed(
+                    local_table_name="sessions_hourly_local",
+                    sharding_key=None,  # TODO: what was the sharding key?
```
Comment (fpacifici) [inline snuba/migrations/snuba_migrations/sessions/0001_sessions.py:136]: Same `org_id`

**Review by fpacifici — COMMENTED** [review]
please see the comments on the sharding key

File: snuba/migrations/snuba_migrations/sessions/0001_sessions.py (hunk 1 above)
Comment (lynnagara) [reply snuba/migrations/snuba_migrations/sessions/0001_sessions.py:127]: Thanks

File: snuba/migrations/snuba_migrations/sessions/0001_sessions.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+                columns=raw_columns,
+                engine=table_engines.Distributed(
+                    local_table_name="sessions_raw_local",
+                    sharding_key=None,  # TODO: what was the sharding key?
+                ),
+            ),
+            operations.CreateTable(
+                storage_set=StorageSetKey.SESSIONS,
+                table_name="sessions_hourly_dist",
+                columns=aggregate_columns,  # TODO: needs verification
```
Comment (lynnagara) [inline snuba/migrations/snuba_migrations/sessions/0001_sessions.py:133]: @fpacifici Do you know if this was changed at all? or the same list of columns was used?

Comment (fpacifici) [reply snuba/migrations/snuba_migrations/sessions/0001_sessions.py:133]: This is the current schema.

```
CREATE TABLE default.sessions_hourly_dist 
(
 `org_id` UInt64, 
  `project_id` UInt64, 
  `started` DateTime, 
   `release` LowCardinality(String), 
  `environment` LowCardinality(String), 
  `duration_quantiles` AggregateFunction(quantilesIf(0.5, 0.9), UInt32, UInt8),
  `sessions` AggregateFunction(countIf, UUID, UInt8), 
  `users` AggregateFunction(uniqIf, UUID, UInt8), 
  `sessions_crashed` AggregateFunction(countIf, UUID, UInt8), 
  `sessions_abnormal` AggregateFunction(countIf, UUID, UInt8), 
  `sessions_errored` AggregateFunction(uniqIf, UUID, UInt8), 
  `users_crashed` AggregateFunction(uniqIf, UUID, UInt8), 
  `users_abnormal` AggregateFunction(uniqIf, UUID, UInt8), 
  `users_errored` AggregateFunction(uniqIf, UUID, UInt8)) 
ENGINE = Distributed(snuba_sessions, default, sessions_hourly_local, org_id)
```

File: snuba/migrations/snuba_migrations/sessions/0001_sessions.py [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+                """,
+            ),
+        ]
+
+    def backwards_local(self) -> Sequence[operations.Operation]:
+        return [
+            operations.DropTable(
+                storage_set=StorageSetKey.SESSIONS, table_name="sessions_raw_local",
+            )
+        ]
```
Comment (fpacifici) [inline snuba/migrations/snuba_migrations/sessions/0001_sessions.py:117]: Why aren't you deleting the materialized view ?

Comment (lynnagara) [reply snuba/migrations/snuba_migrations/sessions/0001_sessions.py]: Thanks, I forgot to add the backwards migrations in

**Review by fpacifici — APPROVED** [review]

**Pull request merged**